src/index.js

```diff
@@ -1 +1,26 @@
 import './style.scss';
+import Request from "./modules/request";
+
+const coinsUrl='https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&order=market_cap_desc&per_page=100&page=1&sparkline=false';
+const url='https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/QzuepOdb4S7f14QxmbA0/scores';
+const request = new Request(coinsUrl);
+const req= new Request(url);
+const coins=document.querySelector('.coins');
```

Comment:
If this query was in the `addToDom`() , function would have less arguments passed into and code would be easier to read.

Reply:
I mean if this line 
`const coins=document.querySelector('.coins');`
was in below function
there wouldnt be need to pass `ul` as function argument
```
const addToDom=async(request,ul)=>{
    var result=await getData(request);
    ul.innerHtml = '';
    console.log(result);
     result.forEach(element => {
             ul.innerHtml +=`<li>${element.name}</li>`;
             console.log(element.name)
          }); 
}
```

Reply:
ok but i think i is better to pass it as argument for code reusability if i want to use it with a different ul or element i wont have to create an other function.